### PR TITLE
Localization updates (2022-06-20)

### PR DESCRIPTION
**Pull request by NeonKnightOA**
* Added localizations for MH-Revenge][-SE
* Fixed a missing version variable in MonsterHunt.*
* Fixed codification for MonsterHunt.est
* Minor fixes

EDIT 22-06-2022: 
* German strings (Thanks @JanUrbansky (eGo) )
* Spanish fixes

### CODE REVIEW

File: resources/System/MonsterHunt.est [hunk 1]
```diff
@@ -5,7 +5,7 @@ Object=(Name={{{package}}}.{{{class}}},Class=Class,MetaClass=Engine.Mutator,Desc
 
 {{#gametypes}}
 Object=(Name={{{package}}}.{{{class}}},Class=Class,MetaClass=Botpack.TournamentGameInfo)
-Preferences=(Caption="{{{est.name}}} v{{{version}}}",Parent="Tipos de Juego",Class={{{package}}}.{{{class}}},Immediate=True)
+Preferences=(Caption="{{{es.name}}} v{{{version}}}",Parent="Game Types",Class={{{package}}}.{{{class}}},Immediate=True)
```
Comment (shrimpza) [inline resources/System/MonsterHunt.est]: This needs to remain `est`; as it aligns with the section names within the `template-options.yml` file.

**Comment by shrimpza** [comment]
This is great, thanks to both of you!

File: resources/System/MonsterHunt.est (hunk 1 above)
Comment (NeonKnightOA) [reply resources/System/MonsterHunt.est]: Oops...

Comment (NeonKnightOA) [reply resources/System/MonsterHunt.est]: Solved.

**Pull request merged**